word_guess.rb

```diff
@@ -0,0 +1,143 @@
+
+require 'colorize'
+
+class NewGame
```

Comment:
I might recommend a name like `Game` or `WordGame`. Since new is generally a keyword in programming, we try to avoid using it in classes and variables.